src/index.ts

```diff
@@ -1,46 +1,122 @@
-import SSO from "./automations/SSO";
 import { JobInfo } from "./common/types";
 import regions from "./common/regions";
 import Job from "./automations/Job";
-import Puppeteer from "puppeteer";
-import { green } from "colors";
+import SSO from "./automations/SSO";
 import { Command, Argument, Option } from "commander";
+import { red } from "colors";
+import ora from "ora";
+// @ts-ignore This can be deleted after https://github.com/enquirer/enquirer/issues/135 is fixed.
+import { Select } from "enquirer";
 
-async function addPosts(jobID: number, regions: string[], cloneFrom: number) {
-    const sso = new SSO();
-    const loginCookies = await sso.login();
-    console.log(green("✓"), "Authentication complete");
+async function getJobFromCLI(job: Job, message: string, spinner: ora.Ora) {
```

Comment:
I think it's better to name it how it's getting the job:
something like `getJobInteractive`

Reply:
:+1: